docs/file-reference.md

```diff
@@ -150,3 +151,11 @@ This is a string in following format: `ContainerPort:ServicePort`
 `ServicePort` is optional. It defaults to `ContainerPort` if not specified.
 
 
+#### type 
+| type | required | possible values         |
+|------|----------|-------------------------|
+|string|    no    | `internal` or `external`|
+
+If `type` is `internal` than port will be accessible only by other containers inside Kubernetes cluster
```

Comment:
You are missing a description for the case when `type` is not specified.

How about putting it into a table? [type, description] (\<unspecified\>, internal, external)